src/html/scan.js

```diff
@@ -13,15 +13,17 @@ function getPwmFormData()
     while (inField = _(`pwm_${ch}_ch`))
     {
         let inChannel = inField.value;
+        let mode = _(`pwm_${ch}_mode`).value;
         let invert = _(`pwm_${ch}_inv`).checked ? 1 : 0;
+        let narrow = _(`pwm_${ch}_nar`).checked ? 1 : 0;
         let failsafeField = _(`pwm_${ch}_fs`);
         let failsafe = failsafeField.value;
         if (failsafe > 2011) failsafe = 2011;
         if (failsafe < 988) failsafe = 988;
         failsafeField.value = failsafe;
 
-        let raw = (invert << 14) | (inChannel << 10) | (failsafe - 988);
-        //console.log(`PWM ${ch} input=${inChannel} fs=${failsafe} inv=${invert} raw=${raw}`);
+        let raw = (narrow << 19) | (mode << 15) | (invert << 14) | (inChannel << 10) | (failsafe - 988);
+        console.log(`PWM ${ch} mode=${mode} input=${inChannel} fs=${failsafe} inv=${invert} nar=${narrow} raw=${raw}`);
```

Comment:
Do we need the debug output?

Reply:
You're right. It was supposed to be commented back out, I'll fix.